### ISSUE TITLE
请问这张图的参数怎么设置呀

### ISSUE THREAD

**Issue by qqkk888** [NJU-PCALab/RAG-Diffusion#19]
我设置了很多组HB_replace、SR_delta，生成效果都不太好
HB_replace = 1，SR_delta = 1
<img width="640" height="320" alt="Image" src="https://github.com/user-attachments/assets/ff49bd41-bb45-46b9-b350-eabf9fe4b30b" />
HB_replace = 1，SR_delta = 0.25
<img width="646" height="326" alt="Image" src="https://github.com/user-attachments/assets/36524c03-4d0b-4a12-933f-556d3ec2d7b8" />
HB_replace = 3，SR_delta = 1
<img width="646" height="323" alt="Image" src="https://github.com/user-attachments/assets/4e0c5e7f-e36d-4136-a1b0-e79b446abea8" />
HB_replace = 7，SR_delta = 1
<img width="646" height="321" alt="Image" src="https://github.com/user-attachments/assets/c0fb0be2-4bb6-4b7a-8963-58c93686ddb6" />

import torch
from RAG_pipeline_flux import RAG_FluxPipeline

pipe = RAG_FluxPipeline.from_pretrained("black-forest-labs/FLUX.1-dev", torch_dtype=torch.bfloat16)
pipe = pipe.to("cuda")

prompt = "On the left, a penguin wearing sunglasses is sunbathing in the desert; in the center, a tiger wearing a scarf is standing on a glacier; on the right, a panda in a windbreaker is walking through the forest."
HB_replace = 3
HB_prompt_list =  [
        "A penguin wearing sunglasses sunbathing in the desert", "A tiger wearing a scarf standing on a glacier", "A panda in a windbreaker walking through the forest"
    ]
HB_m_offset_list =  [
        0.02, 0.35, 0.68
    ]
HB_n_offset_list =  [
        0.1, 0.1, 0.1
    ]
HB_m_scale_list =  [
        0.29, 0.29, 0.29
    ]
HB_n_scale_list = [
        0.8, 0.8, 0.8
    ]
SR_delta = 1
SR_hw_split_ratio = "0.33, 0.33, 0.34"
SR_prompt = "A penguin wearing sunglasses is sunbathing in the scorching desert, the sand glowing brightly around it. BREAK A tiger wearing a scarf stands majestically on a glacier, the crisp cold air visible around its breath as it surveys its icy domain. BREAK A panda in a snug windbreaker walks leisurely through a lush forest, the trees providing a vibrant green backdrop to its path."
height, width = 512, 1024
seed = 1234

image = pipe(
    SR_delta=SR_delta,
    SR_hw_split_ratio=SR_hw_split_ratio,
    SR_prompt=SR_prompt,
    HB_prompt_list=HB_prompt_list,
    HB_m_offset_list=HB_m_offset_list,
    HB_n_offset_list=HB_n_offset_list,
    HB_m_scale_list=HB_m_scale_list,
    HB_n_scale_list=HB_n_scale_list,
    HB_replace=HB_replace,
    seed=seed,
    prompt=prompt,
    height=height,
    width=width,
    num_inference_steps=20,
    guidance_scale=3.5,
).images[0]

filename = "RAG.png"
image.save(filename)
print(f"Image saved as {filename}")
